src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/DiagnosticsClient.cs

```diff
@@ -15,13 +15,18 @@ namespace Microsoft.Diagnostics.NETCore.Client
     /// <summary>
     /// This is a top-level class that contains methods to send various diagnostics command to the runtime.
     /// </summary>
-    public sealed class DiagnosticsClient
```

Comment:
The decision to make this a `sealed` class was intentional. Is there a particular reason you're removing `sealed` here?

Reply:
Good catch. I was looking at deriving from it for the reversed server usage, but don't need that anymore. I'll revert this.